packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionSmoothers_def.hpp

```diff
@@ -208,7 +283,7 @@ void smootherSetup(RCP<Teuchos::ParameterList> params,
     jacobiSetup(params, maxRegPerProc, revisedRowMapPerGrp, regionGrpMats, regionInterfaceScaling);
     break;
   case 2:
-    std::cout << "Gauss-Seidel smoother not implemented yet no smoother is applied" << std::endl;
+    jacobiSetup(params, maxRegPerProc, revisedRowMapPerGrp, regionGrpMats, regionInterfaceScaling);
```

Comment:
Do we need the data computed for the Jacobi preconditioner to be computed in the Gauss-Seidel case?

Reply:
@lucbv @rstumin Right now, it looks like the inverse of the true diagonal needed in Gauss-Seidel is computed during the apply-phase. If we keep it that way, then we don't have to call `jacobiSetup()` here. If we want to pre-compute the inverse of the true diagonal in the setup-phase, then we should 
- rename `jacobiSetup()` to `relaxationSmootherSetup()`
- call `relaxationSmootherSetup()` for both cases, Jacobi and Gauss-Seidel
- use the pre-comuted inverse of the true diagonal during `GSIterate()`

> This would also address @lucbv's [other comment to this PR](https://github.com/trilinos/Trilinos/pull/5701/files#r313500551).

I can have a look at this next week, maybe on Monday. Please let me know what you think.